docs/k0s-multi-node.md

```diff
@@ -24,13 +24,13 @@ The download script accepts the following environment variables:
 
 | Variable                   | Purpose                                           |
 |:---------------------------|:--------------------------------------------------|
-| `K0S_VERSION=v1.23.5+k0s.0 | Select the version of k0s to be installed         |
+| `K0S_VERSION=v1.24.0-beta.0+k0s.0 | Select the version of k0s to be installed         |
```

Comment:
Do we want to re-align the columns in this table again?

Reply:
I wouldn't do it, we know that 1.24.0-beta.0 is just here for a while and we'll switch to 1.24.0 once it's out